tests/components/unifi/test_init.py

```diff
@@ -64,7 +67,7 @@ async def test_controller_no_mac(hass):
 async def test_unload_entry(hass):
     """Test being able to unload an entry."""
     controller = await setup_unifi_integration(hass)
-    assert hass.data[unifi.DOMAIN]
+    assert hass.data[UNIFI_DOMAIN]
 
     assert await unifi.async_unload_entry(hass, controller.config_entry)
-    assert not hass.data[unifi.DOMAIN]
+    assert not hass.data[UNIFI_DOMAIN]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)